portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm

```diff
@@ -64,12 +64,12 @@
         <link href="${pageScriptPath}jquery/cluetip/1.2.10/css/jquery.cluetip.css$!{portalCDNQuery}" rel="stylesheet">
         <link href="${pageWebjarsPath}qtip2/3.0.3/jquery.qtip.min.css$!{portalCDNQuery}" rel="stylesheet">
         <link href="${pageWebjarsPath}pnotify/3.2.1/pnotify.css$!{portalCDNQuery}" rel="stylesheet">
-	<link href="${pageWebjarsPath}pnotify/3.2.1/pnotify.buttons.css$!{portalCDNQuery}" rel="stylesheet">
+        <link href="${pageWebjarsPath}pnotify/3.2.1/pnotify.buttons.css$!{portalCDNQuery}" rel="stylesheet">
         <link href="${pageWebjarsPath}cropper/4.0.0/dist/cropper.min.css$!{portalCDNQuery}" rel="stylesheet">
         <script src="${pageSkinRepo}/${pageSkin}/js/lib/modernizr.js$!{portalCDNQuery}"></script>
+        <script src="${pageWebjarsPath}momentjs/2.22.2/min/moment-with-locales.min.js$!{portalCDNQuery}"></script>
```

Comment:
could we update this to use includeWebjarLibrary as the best thing here is that we don't hardcode the versions other than headscript.js

Reply:
Yep, can do.